strr-web/lang/en.json

```diff
@@ -108,7 +108,9 @@
     "unitInfo": "Rental Unit Information",
     "nickname": "Nickname",
     "businessLicense": "Business License",
-    "platformUrl": "Platform URL",
+    "listingLink": "Listing Link",
+    "listingLinkOptional": "Listing Link (Optional)",
+    "listingLinkHelp": "e.g., airbnb.ca/your_listing123",
```

Comment:
If people enter airbnb.ca/your_listing123 as per the suggestion, they'll get a URL format error. I suggest to make the hint: https://airbnb.ca/your_listing123

Reply:
@andyyanggov please see Roland's comment

Reply:
Updated the design.